pkg/upstream/transformer_test.go

```diff
@@ -13,6 +18,80 @@ func ns(path string) Namespace {
 	return strings.Split(path, "/")
 }
 
+func Test_NewRedirectTransform200(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if data, err := ioutil.ReadFile("testdata/fuse.html"); err == nil {
+			w.WriteHeader(200)
+
+			if _, err := w.Write(data); err != nil {
+				w.WriteHeader(500)
```

Comment:
calling WriteHeader more than once is a bug - in this case since it's in a test we can just use require.NoError